src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -476,13 +471,10 @@ internal void LoadLibraryMigrations(string library)
         {
             string fullLibraryName = library;
 
-            if (!pathManager.ResolveLibraryPath(ref fullLibraryName))
-                return;
-
             string migrationsXMLFile = Path.Combine(Path.GetDirectoryName(fullLibraryName),
                 Path.GetFileNameWithoutExtension(fullLibraryName) + ".Migrations.xml");
 
-            if (!File.Exists(migrationsXMLFile))
+            if (!pathManager.ResolveDocumentPath(ref migrationsXMLFile))
```

Comment:
There are migrations tests on DynamoRevit, but you'll need to have revit installed and RTF (revit test framework) which is here:

https://github.com/DynamoDS/RevitTestFramework